src/analytics.ts

```diff
@@ -353,3 +353,17 @@ export const getCustomerHash = (): Promise<{hash: string}> =>
     postMessageToNativeApp({
         type: 'GET_CUSTOMER_HASH',
     });
+
+export const setTrackingProperty = (
+    system: 'palitagem' | 'medalia',
```

Comment:
@JorgeMucientes I'm adding here, as Abel suggested me, the supported systems that are available. By now for all of them the method is going to return an error, but we will specify since which version they are supported

Reply:
Great. 

BTW it's probably too late as I see this has been merged, but there is a typo. It's "medallia" not "medalia"

Reply:
Sure https://github.com/Telefonica/webview-bridge/pull/52